pkg/apis/v1alpha1/awsnodetemplate.go

```diff
@@ -45,6 +56,10 @@ type AWSNodeTemplateStatus struct {
 	// cluster under the SecurityGroups selectors.
 	// +optional
 	SecurityGroups []SecurityGroupStatus `json:"securityGroups,omitempty"`
+	// AMI contains the current AMI values that are available to the
+	// cluster under the AMI selectors.
+	// +optional
+	AMIs []AMIStatus `json:"ami,omitempty"`
```

Comment:
```suggestion
	AMIs []AMIStatus `json:"amis,omitempty"`
```

This should be plural